### PR TITLE
pass in icon props to presence badge icon slot, and use default strings

### PR DESCRIPTION
**Pull request by micahgodbolt**
Presence badge icon is not getting slot props, therefore you can't override the aria-label. Assuming we [remove the aria-hidden](https://github.com/microsoft/fluentui/pull/22858), this will be necessary to allow for customization of the icon.

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/Avatar-mount_base.html">1095</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/Avatar-mount.html">1085</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/Button-mount_base.html">687</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/Button-mount.html">671</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/FluentProvider-mount_base.html">2287</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/FluentProvider-mount.html">2251</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/FluentProviderWithTheme-mount_base.html">339</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/FluentProviderWithTheme-mount.html">346</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">282</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">297</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">367</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">364</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/MakeStyles-mount_base.html">1877</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22901/perf-test-react-components/MakeStyles-mount.html">1840</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-badge/src/components/PresenceBadge/usePresenceBadge.tsx [hunk 1]
```diff
@@ -37,6 +37,16 @@ const iconMap = (
   }
 };
 
+export const DEFAULT_STRINGS = {
```
Comment (khmakoto) [inline packages/react-badge/src/components/PresenceBadge/usePresenceBadge.tsx:40]: Probably don't want to export these right now

```suggestion
const DEFAULT_STRINGS = {
```

File: packages/react-badge/src/stories/PresenceBadgeStatus.stories.tsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ export const Status = () => {
       <PresenceBadge status="doNotDisturb" />
       <PresenceBadge status="offline" />
       <PresenceBadge status="outOfOffice" />
+      <PresenceBadge icon={{ 'aria-label': 'OOF' }} status="outOfOffice" />
```
Comment (khmakoto) [inline packages/react-badge/src/stories/PresenceBadgeStatus.stories.tsx:14]: I'd probably move this to another story that is specifically about customizing the icon `aria-label` if we want to show this at all.

Comment (spmonahan) [reply packages/react-badge/src/stories/PresenceBadgeStatus.stories.tsx:14]: > if we want to show this at all.

My feeling is that we should not show it here. We're still working out how to consistently handle strings like this for the entire library and this isn't really demonstrating a feature of `PresenceBadge`. It's demonstrating how to use a slot and I think something like this example would be a good addition to the docs section [Customizing Components with Slots](https://61b7c406fa7cd4003a8a07ec-cqsicxkjrr.chromatic.com/?path=/docs/concepts-developer-customizing-components-with-slots--page).

Comment (micahgodbolt) [reply packages/react-badge/src/stories/PresenceBadgeStatus.stories.tsx:14]: yeah, i agree with that. I know sarah has voiced the same concern. There's no reason to have this in any story right now.

Comment (micahgodbolt) [reply packages/react-badge/src/stories/PresenceBadgeStatus.stories.tsx:14]:
```suggestion
```

**Review by spmonahan — APPROVED** [review]

**Pull request merged**